### PR TITLE
[WIP] Fix and enhance useless comments fixer

### LINKED ISSUE #27: useless_comment remove close comment tag
image

### PR DESCRIPTION
**Pull request by PedroTroller**
Closes #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -342,7 +342,7 @@ return $config;
 
 ## PedroTroller/useless_comment
 
-Remove useless comments regarding the method definition
+Remove useless comments regarding the method definition. This fixer is complementary with `phpdoc_trim`, so enable it or use the `@Synfony` rule.
```
Comment (grachevko) [inline README.md]: Synfony -> Symfony

File: src/PedroTroller/CS/Fixer/Comment/UselessCommentFixer.php [hunk 2]
```diff
@@ -12,7 +13,7 @@
     // {@inheritdoc}
     public function getDocumentation()
     {
-        return 'Remove useless comments regarding the method definition';
+        return 'Remove useless comments regarding the method definition. This fixer is complementary with `phpdoc_trim`, so enable it or use the `@Synfony` rule.';
```
Comment (grachevko) [inline src/PedroTroller/CS/Fixer/Comment/UselessCommentFixer.php]: Synfony -> Symfony

Comment (PedroTroller) [reply src/PedroTroller/CS/Fixer/Comment/UselessCommentFixer.php]: source

File: README.md (hunk 1 above)
Comment (PedroTroller) [reply README.md]: source

**Pull request merged**